android/app/src/main/java/app/organicmaps/settings/SettingsPrefsFragment.java

```diff
@@ -373,23 +366,37 @@ else if (preference.getKey() != null && preference.getKey().equals(getString(R.s
   }
 
   // Use this method only on TTS screen.
-  private void initTtsLangInfoLink()
-  {
-    final Preference ttsLangInfoLink = getPreference(getString(R.string.pref_tts_info_link));
-    final String ttsLinkText = getString(R.string.prefs_languages_information_off_link);
-    final Spannable link = new SpannableString(ttsLinkText + "↗");
-    // Set link color.
-    link.setSpan(new ForegroundColorSpan(ContextCompat.getColor(requireContext(),
-                                                                UiUtils.getStyledResourceId(requireContext(), R.attr.colorAccent))),
-                 0, ttsLinkText.length(), 0);
-    ttsLangInfoLink.setSummary(link);
-
-    final String ttsInfoUrl = requireActivity().getString(R.string.tts_info_link);
-    ttsLangInfoLink.setOnPreferenceClickListener(preference -> {
-      final Intent intent = new Intent(Intent.ACTION_VIEW);
-      intent.setData(Uri.parse(ttsInfoUrl));
-      requireContext().startActivity(intent);
-      return false;
+  private void initTtsSupportInfo()
+  {
+    // Init TTS Support link.
+    mTtsPrefSupport = getPreference(getString(R.string.pref_tts_support));
+
+    mTtsPrefSupport.setOnBindTextViewListener(android.R.id.summary, textView -> {
+
+      textView.setMovementMethod(LinkMovementMethod.getInstance());
+
+      String ttsSupportGuideLink = getString(R.string.pref_tts_faq_link);
+      String summaryText = String.format(getString(R.string.pref_tts_support_info),
+                                         ttsSupportGuideLink);
+
+      Spannable summarySpan = new SpannableString(summaryText);
+
+      summarySpan.setSpan(new ClickableSpan() {
+                            @Override
+                            public void onClick(@NonNull View widget) {
+
+                              final String ttsFaqUrl = requireActivity().getString(R.string.tts_faq_url);
+
+                              final Intent intent = new Intent(Intent.ACTION_VIEW);
+                              intent.setData(Uri.parse(ttsFaqUrl));
+                              requireContext().startActivity(intent);
```

Comment:
```suggestion
                              Utils.openUri(requireContext(), Uri.parse(ttsFaqUrl));
```

Reply:
Thanks @Jean-BaptisteC ... I will amend the commit...